src/components/widget.js

```diff
@@ -150,172 +145,168 @@ class Widget extends LitElement {
+              >${this.member.jobTitle}</span
+            >
+
+            <!-- Organization -->
+            ${!!this.member.organization
               ? html`
-            <div class="flex flex-row justify-start items-center mt-1">
-              ${
-                this.member.organization.logo_url &&
-                html`<img
-                  class="mr-1 w-5 h-5"
-                  src="${this.member.organization.logo_url}"
-                />`
+                  <div
+                    class="flex flex-row justify-start items-center mt-1 leading-5"
+                  >
+                    ${this.member.organization.logo_url &&
+                    html`<img
+                      alt=""
+                      class="mr-1 w-5 h-5"
+                      src="${this.member.organization.logo_url}"
+                    />`}
+                    <!-- If organisation doesn't include a protocol (ie https://), add one so it's treated as absolute -->
+                    <a
+                      href="${this.member.organization.website.match(
+                        /https?:\/\//
+                      )
+                        ? this.member.organization.website
+                        : `https://${this.member.organization.website}`}"
+                      target="_blank"
+                      rel="noreferrer"
+                      class="mr-2 text-sm font-semibold text-[#6C4DF6] hover:underline"
+                      >${this.member.organization.name}</a
+                    >
+                    ${this.member.organization.lifecycle_stage === "customer"
+                      ? html`<span class="sr-only">Customer</span>
+                          <span aria-hidden="true"
+                            >${unsafeSVG(iconCustomer)}</span
+                          >`
+                      : nothing}
+                  </div>
+                `
+              : // Just used for spacing when the organisation is not present
+                html`<div class="mt-2"></div>`}
+          </div>
+        </section>
+
+        <!-- Pills -->
+        <section
+          class="flex flex-row justify-start items-center mt-3 space-x-1"
+        >
+          ${this.member.teammate
+            ? html`<obe-pill
+                icon="${iconOrbitLevel}"
+                value="Teammate"
+              ></obe-pill>`
+            : html`<obe-pill
+                icon="${iconOrbitLevel}"
+                name="Orbit"
+                value="${this.member.orbitLevel || "N/A"}"
+              ></obe-pill>`}
+          ${this.member.lastActivityOccurredAt &&
+          html` <obe-pill
+            name="Last Active"
+            value="${formatDate(this.member.lastActivityOccurredAt)}"
+          ></obe-pill>`}
+        </section>
+
+        <hr class="block my-5 border-t border-gray-100" role="none" />
+
+        <!-- Identities -->
+        ${!!this.member.identities &&
+        html`<section>
+          <p class="sr-only">
```

Comment:
Forgot to highlight this in accessibility loom - these headers are kept for screenreaders, since they miss the context clues of seeing logos/the common tag UI from the app